src/plugins/data_view_management/public/components/edit_index_pattern/edit_index_pattern.tsx

```diff
@@ -261,6 +269,20 @@ export const EditIndexPattern = withRouter(
               <EuiSpacer />
               <EuiCallOut title={mappingConflictHeader} color="warning" iconType="warning">
                 <p>{mappingConflictLabel}</p>
+                <EuiLink
+                  href={application.getUrlForApp('management', {
+                    path: `/kibana/dataViews/dataView/${encodeURIComponent(
+                      indexPattern.id!
+                    )}#/?_a=(fieldTypes:!(conflict),tab:indexedFields)`,
```

Comment:
After pressing this link the URL changes but the sync from URL back to app state does not happen. How to implement it?

Also how to properly construct such links?

Reply:
I guess I'd avoid this unless there's a compelling reason why we need to have use url based navigation for a management action.

Reply:
I looked into this, and if we want to avoid doing a refactoring of the management state, we could add a simple ref API to the tabs component that currently manages the state. And to avoid manually constructing the URL state, we can use the `setStateToKbnUrl` utility.

I created a PR here including both changes if we're interested in this approach: https://github.com/jughosta/kibana/pull/8.